tests/ui/README.md

```diff
@@ -0,0 +1,104 @@
+# Integration Tests for the [Mozilla Addons Website][amo].
+## How to run the tests locally
+### Clone the repository
+
+If you have cloned this project already then you can skip this, otherwise you'll
+need to clone this repo using Git. If you do not know how to clone a GitHub
+repository, check out this [help page][git-clone] from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining
+them in the future, it would be a good idea to create a fork of this repository
+first, and then clone that. GitHub also has great instructions for
+[forking a repository][git-fork].
+
+### Server Install
+
+Follow the instructions found [here][addons-server-docs].
+
+### Run the tests
+
+Included in the docker-compose file is an image containing firefox nightly. [tox][Tox]
+is our test environment manager and [pytest][pytest] is the test runner.
+
+To run the tests, execute the command below:
+```sh
+docker-compose exec --user root selenium-firefox tox -e ui-tests
+```
```

Comment:
Let's add the warning about deleting the database here again. Just in case someone will only look at this document and not at the other.